README.rst

```diff
@@ -21,6 +21,25 @@ Enrique Garcia, garcia [at] lapp.in2p3.fr
 Lukas Nickel, lukas.nickel [at] tu-dortmund.de
 
 
+Requesting a MC analysis
+------------------------
+As a LST member, you may require a MC analysis with a specific configuation, for exemple to later analyse a specific source with tuned MC parameters.      
+
+To do so, please:
+
+#. open a pull request in lstMCpipe, adding the desired configuration in a new directory in `production_configs`
+#. you may have a look at the `production_configs/20220222_template/` as an example
+#. Add the date and a descriptive production ID name to you directory and config
+#. the requested config must contain:
+
+* a lstchain config file
```

Comment:
* a lstchain config file. 
    - Please note that the lstchain configuration must contain the key:entries for **all** the lstchain stages (r0_to_dl1 or dl1ab, training and dl1_to_dl2)

Reply:
??
you mean lstmcpipe config?

Reply:
I mean that, f. eg, this configuration https://github.com/cta-observatory/cta-lstchain/blob/master/lstchain/data/lstchain_dl1ab_tune_MC_to_Crab_config.json won't be valid. This configuration only contains the  dl1ab lstchain configuration items, and lstmcpipe only accepts a single lstchain file. We woudl need a file containing the information for all the rest of the stages (RF features, integrator and tailcuts for the r0_dl1 stage, cuts for the dl2+ stages....)

Reply:
This is a valid lstchain config
the parts of the config that are not explicitly provided are treated with the default config by lstchain.

Reply:
Although I agree, an exhaustive explicit config might be better